src/views/Simulation/utils/MapToolBarContainer.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import { Container } from 'pixi.js';
+import { createSprite } from '../components/Scene/pixiUtils';
+
+export class MapToolBar extends Container {
+  constructor(mapContainer, toolBarTextures) {
```

Comment:
"toolbar" is a single word, we should probably rename these:
- MapToolBarContainer.js -> MapToolbarContainer.js
- MapToolBar -> MapToolbar
- toolBarTextures -> toolbarTextures